stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/natives.bal

```diff
@@ -280,6 +280,7 @@ public struct Retry {
 @Field {value:"validateCertEnabled: The status of validateCertEnabled {default value : false (disable)}"}
 @Field {value:"cacheSize: Maximum size of the cache"}
 @Field {value:"cacheValidityPeriod: Time duration of cache validity period"}
+@Field {value:"hostNameVerificationEnabled: The status of hostNameVerificationEnabled {default value : true (enable)}"}
```

Comment:
Instead of repeating the field name in description shall we provide a meaningful description?